### PR TITLE
SEATUNNEL-784: add FAQ document

### PR DESCRIPTION
**Pull request by yx91490**
https://github.com/apache/incubator-seatunnel/issues/784

### CODE REVIEW

**Comment by yx91490** [comment]
@wolfboys Hi, I need your help to correct the files path.

File: faq/faq.md [hunk 1]
```diff
@@ -0,0 +1,348 @@
...
+sink {
+	elasticsearch { ... }
+}
+```
+
+**FAQ 7.** Are there any HBase plugins?
+
+There is hbase input plugin, download it from here：https://github.com/garyelephant/waterdrop-input-hbase
+
+**FAQ 8.** How to use SeaTunnel to write data to Hive?
```
Comment (RickyHuo) [inline faq/faq.md:92]: SeaTunnel has implemented `Hive` output plugin. We can notice this.

File: faq/faq.md [hunk 2]
```diff
@@ -0,0 +1,348 @@
+---
+title: SeaTunnel FAQ
+---
+
+**FAQ 1.** I encounter a problem when using SeaTunnel and I cannot solve it by myself. What should I do?Firstly search in [Issue list](https://translate.google.com/translate?hl=zh-CN&prev=_t&sl=auto&tl=en&u=https://github.com/apache/incubator-seatunnel/issues) or [mailing list](https://translate.google.com/translate?hl=zh-CN&prev=_t&sl=auto&tl=en&u=https://lists.apache.org/list.html%3Fdev%40seatunnel.apache.org) to see if someone has already asked the same question and got the answer. If you still cannot find the answer, you can contact community members for help in[ these ways](https://translate.google.com/translate?hl=zh-CN&prev=_t&sl=auto&tl=en&u=https://github.com/apache/incubator-seatunnel%23contact-us#contact-us) .
```
Comment (dailidong) [inline faq/faq.md:5]: Please replace `https://translate.google.com/translate?hl=zh-CN&prev=_t&sl=auto&tl=en&u=https://github.com/apache/incubator-seatunnel/issues` with  `https://github.com/apache/incubator-seatunnel/issues`

Comment (yx91490) [reply faq/faq.md]: thanks, fixed

**Comment by wolfboys** [comment]
> 

👌🏻

**Comment by wolfboys** [comment]
> @wolfboys Hi, I need your help to correct the files path.

👌🏻

**Comment by yx91490** [comment]
Many thanks for your help  @wolfboys

**Comment by wolfboys** [comment]
LGTM.

File: faq/faq.md [hunk 3]
```diff
@@ -0,0 +1,348 @@
...
+     ...
+    }
+    sink {
+      ...
+    }
+    ```
+
+参考的log4j配置文件内容如下：
+
+```
```
Comment (CalvinKirs) [inline faq/faq.md:264]: hi,this has not been modified to english

File: faq/faq.md [hunk 4]
```diff
@@ -0,0 +1,348 @@
...
+In SeaTunnel, the data type will not be actively converted. After the Input reads the data, the corresponding
+
+Schema. When writing ClickHouse, the field type needs to be strictly matched, and the mismatch needs to be done.
+
+Data conversion, data conversion can be achieved through the following two plug-ins:
+
+1. Filter Convert plugin
+2. Filter Sql plugin
+
+Detailed data type conversion reference: [https://interestinglab.github.io/seatunnel-docs/#/zh-cn/v1/configuration/output-plugins/Clickhouse?id=clickhouse%e7%b1%bb%e5%9e%8b%e5%af%b9%e7%85%a7%e8%a1%a8](https://interestinglab.github.io/seatunnel-docs/#/zh-cn/v1/configuration/output-plugins/Clickhouse?id=clickhouse类型对照表)
```
Comment (RickyHuo) [inline faq/faq.md:311]: Reference https://interestinglab.github.io/seatunnel-docs/#/en/configuration/output-plugins/Clickhouse?id=clickhouse-data-type-check-list would be better.

Comment (yx91490) [reply faq/faq.md]: fixed

**Review by RickyHuo — APPROVED** [review]
LGTM

**Comment by wolfboys** [comment]
good job.👍

**Pull request merged**